container_images/daisy-builder/main.sh

```diff
@@ -81,6 +83,24 @@ function generate_build_workflow() {
   echo -e "$config" > "$WF"
 }
 
+tag_commit() {
+  LATEST_VERSION=$(generate_new_version)
+  if [[ "$LATEST_VERSION" != "$VERSION" ]]; then
+    echo "not valid build $LATEST_VERSION != $VERSION"
+    return 1
+  fi
+  TAGGER_CMD="/tagger --token-file-path=${GITHUB_ACCESS_TOKEN} --tag=${LATEST_VERSION} --sha=${PULL_BASE_SHA} "
+  TAGGER_CMD+="--org=${REPO_OWNER} --repo=${REPO_NAME}"
```

Comment:
suggest escape the newline, rather than +=

Reply:
done